test/unittest/tests/test_prepare_mod_sort.cpp

```diff
@@ -256,10 +256,12 @@ TEST_CASE("B-tree ID sort test", "[mod_compare]")
     const int key_count = 1;
     bool ret;
     WT_ITEM *keys[key_count];
+    std::string key_str = "1";
 
     allocate_key_space(session, key_count, keys);
 
-    init_key(keys[0], "1");
+    WT_ASSERT(session, __wt_buf_init(session, keys[0], 1) == 0);
+    WT_ASSERT(session, __wt_buf_set(session, keys[0], "1", key_str.size()) == 0);
```

Comment:
Why not call `init_key(session, keys[0], key_str)` like below?

Reply:
Must've missed that one, thanks!